interop_binaries/src/testcontainer.rs

```diff
@@ -1,24 +1,90 @@
 //! Provides a testcontainer image to allow container-based testing of Janus.
 
-use crate::load_zstd_compressed_docker_image;
+use crate::test_util::load_zstd_compressed_docker_image;
 use std::sync::Mutex;
 use testcontainers::{core::WaitFor, Image};
 
 // interop_aggregator.tar.zst is created by this package's build.rs.
+const INTEROP_CLIENT_IMAGE_BYTES: &[u8] =
+    include_bytes!(concat!(env!("OUT_DIR"), "/interop_client.tar.zst"));
+static INTEROP_CLIENT_IMAGE_HASH: Mutex<Option<String>> = Mutex::new(None);
+
 const INTEROP_AGGREGATOR_IMAGE_BYTES: &[u8] =
     include_bytes!(concat!(env!("OUT_DIR"), "/interop_aggregator.tar.zst"));
 static INTEROP_AGGREGATOR_IMAGE_HASH: Mutex<Option<String>> = Mutex::new(None);
 
+const INTEROP_COLLECTOR_IMAGE_BYTES: &[u8] =
+    include_bytes!(concat!(env!("OUT_DIR"), "/interop_collector.tar.zst"));
+static INTEROP_COLLECTOR_IMAGE_HASH: Mutex<Option<String>> = Mutex::new(None);
+
+/// Represents a Janus Client as a testcontainer image.
+#[non_exhaustive]
+pub struct Client {}
+
+impl Client {
+    // The internal port that the Client serves on.
```

Comment:
This may as well be a doc comment.
```suggestion
    /// The internal port that the Client serves on.
```